internal/pipeline/pipeline.go

```diff
@@ -60,6 +73,69 @@ type Pipeline struct {
 	Validators []Function `yaml:"validators,omitempty"`
 }
 
+// String returns the string representation of Pipeline
+func (p *Pipeline) String() (string, error) {
```

Comment:
hmm...It isn't expected from a stringer method to return an error. Generally people use it as part of fmt.Printf(...), but now caller has to worry about handling this error and they don't even know what to do if this fails :)

May be we can default to just printing the name of pipeline ? instead of returning an error.